qiita_db/handlers/prep_template.py

```diff
@@ -63,3 +66,16 @@ def get(self, prep_id):
             response = {'data': pt.to_dataframe().to_dict(orient='index')}
 
         self.write(response)
+
+
+class PrepTemplateAPItestHandler(OauthBaseHandler):
+    @authenticate_oauth
+    def post(self):
+        prep_info_dict = loads(self.get_argument('prep_info'))
```

Comment:
Is the size of the metadata something we should be concern about? It seems like if we have a large study we can hit the maximum parameter size really quickly.